scripts/install_update_linux.sh

```diff
@@ -31,5 +31,5 @@ echo $GITHUB_URL
 # install/update the local binary
 curl -L -o dblab.tar.gz $GITHUB_URL
 tar xzvf dblab.tar.gz dblab
-mv -f dblab "$DIR"
+mv -f dblab "$DIR/dblab"
```

Comment:
assumes `$DIR` is a pre-existing writable directory?  (which in my case it isn't, not even `~/.local` exists)

Reply:
Yeah, it does, but you can overwrite it to whatever you want.


```sh
$ DIR="/path/to/other/location" ./scripts/install_update_linux.sh
```